src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from enum import Enum
+from typing import Dict, List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from serenity_types.refdata.options import OptionType
+
+
+class VolModel(Enum):
+    """
+    Currently supported volatility models.
+    """
+
+    SVI = "SVI"
+    """
+    Stochastic volatility (SVI) calibrated volatility model.
+    """
+
+    BLACK_SCHOLES = "BLACK_SCHOLES"
+    """
+    Classic Black-Scholes volatility model.
+    """
+
+
+class VolSurfaceType(Enum):
+    """
+    Representation of the volatility surface, e.g. raw inputs vs. fitted surface.
+    """
+
+    RAW = "RAW"
+    """
+    A volatility surface containing the raw inputs: the option prices, IV's, etc..
+    """
+
+    FITTED = "FITTED"
```

Comment:
I guess model need to be updated from INTERPOLATED to FITTED then?

Reply:
What do you mean? "model" here?

Reply:
Current Kevin's code is expecting VolSurfaceType "RAW" or "INTERPOLATED"

Reply:
I will make the specification consistent and use INTERPOLATED, same as @kmgivens.